src/main/java/core/basesyntax/file/FileReaderImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.file;
+
+import exception.FileException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) {
+        List<String> reportList;
+        try {
+            reportList = Files.readAllLines(Path.of(filePath));
+        } catch (IOException e) {
+            throw new FileException("Can't read data from file: " + filePath);
```

Comment:
Consider including the original exception `e` as a cause in the `FileException` to provide more context about the error. For example: `throw new FileException("Can't read data from file: " + filePath, e);`